diff -Nurp php-7.1.16.orig/sapi/litespeed/lsapilib.c php-7.1.16/sapi/litespeed/lsapilib.c
--- php-7.1.16.orig/sapi/litespeed/lsapilib.c	2018-03-28 02:12:02.000000000 -0500
+++ php-7.1.16/sapi/litespeed/lsapilib.c	2018-03-30 08:54:33.695677870 -0500
@@ -755,7 +755,7 @@ static int (*fp_lve_leave)(struct liblve
 static int (*fp_lve_jail)( struct passwd *, char *) = NULL;
 static int lsapi_load_lve_lib(void)
 {
-    s_liblve = dlopen("liblve.so.0", RTLD_LAZY);
+    s_liblve = dlopen("liblve.so.0", RTLD_NOW);
     if (s_liblve)
     {
         fp_lve_is_available = dlsym(s_liblve, "lve_is_available");
diff -Nurp php-7.1.16.orig/Zend/zend_portability.h php-7.1.16/Zend/zend_portability.h
--- php-7.1.16.orig/Zend/zend_portability.h	2018-03-28 02:11:55.000000000 -0500
+++ php-7.1.16/Zend/zend_portability.h	2018-03-30 08:55:44.091847281 -0500
@@ -146,11 +146,11 @@
 # endif
 
 # if defined(RTLD_GROUP) && defined(RTLD_WORLD) && defined(RTLD_PARENT)
-#  define DL_LOAD(libname)			dlopen(libname, RTLD_LAZY | RTLD_GLOBAL | RTLD_GROUP | RTLD_WORLD | RTLD_PARENT)
+#  define DL_LOAD(libname)			dlopen(libname, RTLD_NOW  | RTLD_GLOBAL | RTLD_GROUP | RTLD_WORLD | RTLD_PARENT)
 # elif defined(RTLD_DEEPBIND) && !defined(__SANITIZE_ADDRESS__)
-#  define DL_LOAD(libname)			dlopen(libname, RTLD_LAZY | RTLD_GLOBAL | RTLD_DEEPBIND)
+#  define DL_LOAD(libname)			dlopen(libname, RTLD_NOW  | RTLD_GLOBAL | RTLD_DEEPBIND)
 # else
-#  define DL_LOAD(libname)			dlopen(libname, RTLD_LAZY | RTLD_GLOBAL)
+#  define DL_LOAD(libname)			dlopen(libname, RTLD_NOW  | RTLD_GLOBAL)
 # endif
 # define DL_UNLOAD					dlclose
 # if defined(DLSYM_NEEDS_UNDERSCORE)
